test/scenario/metering-plans/standard-services-hours/src/test/test.js

```diff
@@ -0,0 +1,171 @@
+'use strict';
+
+const moment = require('abacus-moment');
+const oauth = require('abacus-oauth');
+const request = require('abacus-request');
+const util = require('util');
+const uuid = require('uuid');
+
+const debug = require('abacus-debug')('abacus-service-hours-scenario-test');
+
+const doGet = util.promisify(request.get);
+const doPost = util.promisify(request.post);
+
+const env = {
+  secured: process.env.SECURED === 'true',
+  systemClientId: process.env.CLIENT_ID,
+  systemClientSecret: process.env.CLIENT_SECRET,
+  authServerURL: process.env.AUTH_SERVER_URL || 'http://localhost:9882',
+  collectorURL: process.env.COLLECTOR_URL || 'http://localhost:9080',
+  reportingURL: process.env.REPORTING_URL || 'http://localhost:9088',
+  provisioningURL: process.env.PROVISIONING_URL || 'http://localhost:9880',
+  pollInterval: process.env.POLL_INTERVAL || 300
+};
+
+describe('standard services hours scenario test', () => {
+  const timestamp = moment.now();
+  const hourInMillis = 1 * 60 * 60000;
+  const resourceId = 'sampler-test-resource-id';
+  const meteringPlanId = 'standard-services-hours';
+  const planName = 'standard';
+
+  let usageDoc;
+  let orgID;
+  let systemToken;
+
+  const authHeader = (token) => token ? { authorization: token() } : {};
+
+  const buildUsageDoc = (orgID, hours) => {
+    const usageDoc = {
+      start: timestamp,
+      end: timestamp,
+      organization_id: orgID,
+      dedup_id: uuid.v4(),
+      space_id: 'sampler-space-id',
+      consumer_id: 'sampler-consumer-id',
+      resource_id: resourceId,
+      plan_id: planName,
+      resource_instance_id: 'sampler-resource-instance-id',
+      measured_usage: [{
+        measure: 'duration',
+        quantity: hourInMillis * hours
+      }]
+    };
+
+    return usageDoc;
+  };
+
+  const sleep = (duration) => {
```

Comment:
The `sleep` and `eventually` methods are already used in the `dedup` test, unless I am mistaken. Consider a way extract these somewhere (maybe in our mocha wrapper) and make them reusable.